homeassistant/components/zha/__init__.py

```diff
@@ -109,8 +109,8 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     device_registry = await hass.helpers.device_registry.async_get_registry()
     device_registry.async_get_or_create(
         config_entry_id=config_entry.entry_id,
-        connections={(CONNECTION_ZIGBEE, str(zha_gateway.application_controller.ieee))},
-        identifiers={(DOMAIN, str(zha_gateway.application_controller.ieee))},
+        connections={(CONNECTION_ZIGBEE, str(zha_gateway.application_controller.ieee))},  # type: ignore[attr-defined]
+        identifiers={(DOMAIN, str(zha_gateway.application_controller.ieee))},  # type: ignore[attr-defined]
```

Comment:
The type ignores will remove themselves once `ZHAGateway.application_controller` is typed in one of the next PRs.